Cargo.toml

```diff
@@ -124,7 +124,7 @@ etcd-client = "0.13"
 fst = "0.4.7"
 futures = "0.3"
 futures-util = "0.3"
-greptime-proto = { git = "https://github.com/GreptimeTeam/greptime-proto.git", rev = "ec801a91aa22f9666063d02805f1f60f7c93458a" }
+greptime-proto = { git = "https://github.com/GreptimeTeam/greptime-proto.git", rev = "7b7fa439c68929e7f03cbc53856a14de74cb9bc1" }
```

Comment:
Do we have a related PR?

Reply:
wait for https://github.com/GreptimeTeam/greptime-proto/pull/209/